setup.cfg

```diff
@@ -37,33 +37,35 @@ select =
 
 [mypy]
 ignore_missing_imports = True
-
-[mypy-cudf._lib.*]
+# We set --explicit-package-bases --namespace-packages in pre-commit
+# which means that mypy thinks the packages are called python.foo.foo
+# hence the leading wildcard
+[mypy-*.cudf._lib.*]
```

Comment:
Why do we need to set those options now if it's causing this extra difficulty?

Reply:
https://github.com/rapidsai/cudf/pull/11640#discussion_r960878090